src/main/java/tic/tac/toe/Board.java

```diff
@@ -4,26 +4,40 @@
 public class Board {
```

Comment:
I think this class is doing too much; `Board` class should only be responsible for anything related to `Board`

Reply:
cleaned it up, moved non-board related logic to game